dashboards/kv-dashboard.json

```diff
@@ -0,0 +1,115 @@
+{
+  "_base": "dashboard",
+  "title": "2. Data Service Dashboard",
```

Comment:
Can you remove this for now? It looks a little odd as no other dashboards are numbered. I agree that it might be nice to order the dashboards in some way rather than have the unordered as they currently are, but I haven't thought exactly how we might do that. Might be good to discuss options. At any rate, for now would be best to remove the number.